packages/inter-protocol/src/psm/psm.js

```diff
@@ -212,7 +218,10 @@ export const prepare = async (zcf, privateArgs, baggage) => {
     // one immediately below. This `burnMinted`
     // happen only if the `atomicRearrange` does *not* throw.
     burnMinted(afterFee);
-    totalAnchorProvided = AmountMath.add(totalAnchorProvided, maxAnchor);
+    baggage.set(
+      'totalAnchorProvided',
+      AmountMath.add(baggage.get('totalAnchorProvided'), maxAnchor),
```

Comment:
I can see why you were asking about record-like access to baggage. It's tempting to use a proxy. But I guess that's what exo does.